cmd/mailslurper/config.json

```diff
@@ -1,16 +1,18 @@
 {
 	"wwwAddress": "localhost",
-	"wwwPort": 8080,
+	"wwwPort": 8081,
 	"serviceAddress": "localhost",
 	"servicePort": 8085,
+	"serviceURL": "",
 	"smtpAddress": "localhost",
 	"smtpPort": 2500,
 	"dbEngine": "SQLite",
 	"dbHost": "",
 	"dbPort": 0,
-	"dbDatabase": "./mailslurper.db",
+	"dbDatabase": "/tmp/mailslurper.db",
 	"dbUserName": "",
 	"dbPassword": "",
+	"dbEphemeral": false,
```

Comment:
This is also changing current default behavior. Currently the SQLite DB is recreated on restart of the application. I think this should default to true (assuming this means it will drop and recreate), and documentation should be added around this new feature. I like the idea though!